### PR TITLE
Test Blog Post

### PR DESCRIPTION
**Pull request by michaelwnau**
This PR contains a test blog post to troubleshoot OpenGraph meta tags.

### CODE REVIEW

**Comment by michaelwnau** [comment]
Screenshot 2023-10-25 084115

**Comment by michaelwnau** [comment]
Screenshot 2023-10-25 084520

**Review by mbarger14 — APPROVED** [review]

File: _posts/2023/01/2023-01-01-how-to-implement-opengraph.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - opengraph
+title: "How To Implement OpenGraph: A Simple Guide"
+type: Article
+Image: /assets/images/logos/default_og_wh.png
+Description: "A simple guide to implementing OpenGraph on your website."
+Author: zCore Group
+---
+
+## What is OpenGraph?
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod, nisl quis tincidunt aliquam, nunc nisl aliquet nunc, quis ultricies nisl nunc vitae nunc. Sed euismod, nisl quis tincidunt aliquam, nunc nisl aliquet nunc, quis ultricies nisl nunc vitae nunc. Sed euismod, nisl quis tincidunt aliquam, nunc nisl aliquet nunc, quis ultricies nisl nunc vitae nunc. Sed euismod, nisl quis tincidunt aliquam, nunc nisl aliquet nunc, quis ultricies nisl nunc vitae nunc.
```
Comment (bastosmichael) [inline _posts/2023/01/2023-01-01-how-to-implement-opengraph.md:19]: you've got Lorem Ipsum's buddy

**Review by bastosmichael — CHANGES_REQUESTED** [review]
please remove the place holders and replace with article.

File: _posts/2023/01/2023-01-01-how-to-implement-opengraph.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+post_image: /assets/images/blog/motivation.jpeg
+tags:
+  - tech
+  - app
+  - opengraph
+title: "How To Implement OpenGraph: A Simple Guide"
+type: Article
+Image: /assets/images/logos/default_og_wh.png
+Description: "A simple guide to implementing OpenGraph on your website."
+Author: zCore Group
```
Comment (bastosmichael) [inline _posts/2023/01/2023-01-01-how-to-implement-opengraph.md:15]: Feel free to include yourself

File: _posts/2023/01/2023-01-01-how-to-implement-opengraph.md (hunk 1 above)
Comment (michaelwnau) [reply _posts/2023/01/2023-01-01-how-to-implement-opengraph.md:19]: Sorry - just to make it clear: I wasn't writing an article here, only trying to troubleshoot the meta tags against the LI post-inspector. I can turn it into an article later but the intent here is to test the meta tags against the new front matter, troubleshoot, and then reformat our front matter based on the results.

File: _posts/2023/01/2023-01-01-how-to-implement-opengraph.md (hunk 2 above)
Comment (michaelwnau) [reply _posts/2023/01/2023-01-01-how-to-implement-opengraph.md]: It is in the `author` but I am testing `Author` 

Screenshot 2023-10-25 090308

**Comment by tfreebern2** [comment]
@michaelwnau Should I be seeing the new Blog post in the local build?

**Pull request closed without merging**